### DISCUSSION TITLE
Feature Request: More specific category permissions

### DISCUSSION THREAD

**Discussion by ghost** [craftcms/cms#8648] [Ideas]
It would be nice to have more specific permissions for categories. Sections allow you to choose which groups/users can create, edit and delete entries for that section. I suggest that category groups get those same permission options.

Thats all, thanks in advance!

**Comment by KatieMFritz** [comment]
+1. Currently you can only grant full access to category groups. I want users to be able to only select from existing categories or create new ones (not edit or delete existing categories).

Where is this on the roadmap? Deciding whether we can implement a CP access strategy.

**Comment by michaelcolletto** [comment]
Yes, please! It would be greatly helpful if Categories shared the same permission options as Entries: e.g.,
![category permissions mockup](https://user-images.githubusercontent.com/43583917/61883354-4a9ce800-aec8-11e9-894b-41445a8bde1f.png)

**Comment by davidhellmann** [comment]
We don't use categories since some  weeks. We do all now with entries. Brandon said categories are die in the near future and `entries` becomes `content` and there is all the stuff in one place.
